### PR TITLE
fix javadocs for telemetry core

### LINKED ISSUE #236: Fix javadoc warnings in telemetry_core
We have a lot of warnings emitted when running `./gradlew javadoc` in the `telemetry_core` module:

```
[snip]
/Users/jplumb/code/newrelic-telemetry-sdk-java/telemetry_core/src/main/java/com/newrelic/telemetry/SpanBatchSenderFactory.java:27: warning: no @return
  default SpanBatchSender createBatchSender(String apiKey) {
                          ^
/Users/jplumb/code/newrelic-telemetry-sdk-java/telemetry_core/src/main/java/com/newrelic/telemetry/SpanBatchSenderFactory.java:54: warning: no @param for apiKey
  default SenderConfigurationBuilder configureWith(String apiKey) {
                                     ^
/Users/jplumb/code/newrelic-telemetry-sdk-java/telemetry_core/src/main/java/com/newrelic/telemetry/SpanBatchSenderFactory.java:54: warning: no @return
  default SenderConfigurationBuilder configureWith(String apiKey) {
                                     ^
78 warnings
```

This could certainly be tackled with multiple PRs.

### PR DESCRIPTION
**Pull request by uaihebert**
fix #236

### CODE REVIEW

**Comment by uaihebert** [comment]
After this PR gets merged, I will work on the last one.

File: telemetry_core/src/main/java/com/newrelic/telemetry/events/Event.java [hunk 1]
```diff
@@ -22,6 +22,7 @@
    *
    * @param eventType The type of event.
    * @param attributes The key-value pairs that make up the event.
+   * @param timestamp time stamp in epoch
```
Comment (breedx-nr) [inline telemetry_core/src/main/java/com/newrelic/telemetry/events/Event.java:25]: millis?

**Review by breedx-nr — CHANGES_REQUESTED** [review]
Looks good, let's just change that one timestamp to clarify units.  Thanks again!

**Comment by breedx-nr** [comment]
(Also needs a rebase)

**Comment by uaihebert** [comment]
all done. ( :

**Review by breedx-nr — APPROVED** [review]

**Comment by breedx-nr** [comment]
Thanks again @uaihebert !

**Pull request merged**